internal/db/db.go

```diff
@@ -339,13 +339,20 @@ func (db *db) DeleteDocActorRelationship(
 	return client.DeleteDocActorRelationshipResult{RecordFound: recordFound}, nil
 }
 
-func (db *db) GetNodeIdentity(context.Context) (immutable.Option[identity.PublicRawIdentity], error) {
+func (db *db) GetNodeIdentity(_ context.Context) (immutable.Option[identity.PublicRawIdentity], error) {
```

Comment:
Because just `context.Context` seems like a mistake. The underscore says that the function parameters are this way because we're following some interface pattern or something like that but we aren't using this parameter within the function.

Reply:
This is a known go idiom?

Reply:
It is the same as the other areas we use `_` to ignore a returned argument. https://go.dev/ref/spec#Blank_identifier

The only reason why just using `context.Context` worked is because it was the only parameter in the method. If there was more than one, it would be a syntax error.

Unused function parameters are allowed in Go and that's why we often have `ctx context.Context` as parameter with `ctx` never being used. I feel that's misleading. Doing `_ context.Context` at least tells the reader that this parameter needed to be there but we aren't using it.